RecoPixelVertexing/PixelVertexFinding/plugins/PixelVertexProducer.cc

```diff
@@ -1,4 +1,4 @@
-#include "RecoPixelVertexing/PixelVertexFinding/interface/PixelVertexProducer.h"
+#include "RecoPixelVertexing/PixelVertexFinding/plugins/PixelVertexProducer.h"
```

Comment:
please take this update as an opportunity to merge PixelVertexProducer.h contents into .cc file, which is a recommended approach

Reply:
implemented this in 356519de19f